coordinator/internal/controller/api/controller.go

```diff
@@ -28,6 +28,17 @@ func InitController(cfg *config.Config, chainCfg *params.ChainConfig, db *gorm.D
 
 	log.Info("verifier created", "openVmVerifier", vf.OpenVMVkMap)
 
+	// TODO: enable this when the libzkp has been updated
```

Comment:
When would this code and l2geth endpoint in the config be used?

Reply:
in the following PR, which has completed the implement of libzkp

Reply:
got it and thanks!